client/state/products-list/schema.js

```diff
@@ -0,0 +1,33 @@
+export const productsListSchema = {
+	type: 'object',
+	required: [],
+	patternProperties: {
+		'.+': {
+			type: 'object',
+			required: [
+				'product_id',
+				'product_name',
+				'product_slug',
+				'description',
+				'cost',
+				'is_domain_registration',
+				'cost_display' ],
+			properties: {
+				product_id: { type: 'integer' },
+				product_name: { type: 'string' },
+				product_slug: { type: 'string' },
+				description: { type: 'string' },
+				cost: { type: 'integer' },
+				prices: {
```

Comment:
Unless we use the data in the prices array (which I think we don't) we don't need to be super strict here. We can just expect an object.